rclpy/rclpy/logging.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from enum import IntEnum
+
+import rclpy.impl.logger_rcutils
+
+
+class LoggingSeverity(IntEnum):
+    """
+    Enum for logging severity levels.
+
+    This enum must match the one defined in rcutils/logging.h
+    """
+
+    DEBUG = 0
+    INFO = 1
+    WARN = 2
+    ERROR = 3
+    FATAL = 4
+
+
+def get_named_logger(name):
+    return rclpy.impl.logger_rcutils.RcutilsLogger(name)
```

Comment:
It looks strange to me to have `logger_rcutils` and then `RcutilsLogger` next to it. Why is this not called `rcutils_logger.RcutilsLogger`?

Reply:
I imagined there might be other logger implementations and it would be easier if the files were grouped together alphabetically, but we are _so_ far from having that situation! updated in c8807c56c3e6461dd8f8ebfb032dfa497a1aa193